### PR TITLE
Feature/movie page styling

### PR DESCRIPTION
**Pull request by TimNguyen21**
## Type of change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Styling- no new features
- [x] Refactoring- no new features

## Detailed Description
- Added styling for the movie details page

## Why is this change required? What problem does it solve?
- Add styling for UX

## Files modified:
- movieDetails.js
- movieDetails.scss

### CODE REVIEW

File: src/components/MovieDetails/MovieDetails.scss [hunk 1]
```diff
@@ -1,13 +1,45 @@
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 1em;
+  margin-top: 7em;
+  img:first-child {
+    width: 20em;
+    margin: .5em 0em;
+  }
+  img:nth-child(2) {
```
Comment (CLPena) [inline src/components/MovieDetails/MovieDetails.scss:32]: This is a really clever way of solving the problem of having a dynamically created background. Love the clear and reusable logic!

**Comment by CLPena** [comment]
This looks great! The page looks fantastic, too!

**Comment by hborrelli1** [comment]
Looks good Tim!

**Comment by megan-venetianer** [comment]
This looks awesome!

**Pull request merged**